demo/AutoLoader.php

```diff
@@ -0,0 +1,188 @@
+<?php
```

Comment:
As far as I could see the original demo had an in-line loader and the switch to PSR4 style loading only happened after composer had replaced the various options used prior to that? One of the reasons I had a problem initially.